cmd/repo/add.go

```diff
@@ -47,7 +47,7 @@ Adds a repository to the system. URLs, local files or inline repo can be specifi
 
 # Inline:
 
- luet repo add testfo --description "Bar" --url "FOZZ" --type "ff"
+ luet repo add testfo --description "Bar" --url "FOZZ" --type "ff" --username "user" --passwd "password"
```

Comment:
To demonstrate best practices, maybe this is better:

```suggestion
 luet repo add testfo --description "Bar" --url "FOZZ" --type "ff" --username "user" --passwd $(echo "$PASSWORD")
```

Generally, people shouldn't pass passwords in a way that it's written in bash history. Environment variables can be set in various ways, without running a command explicitly (which would have stored it in bash history too).